.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ['3.10'] # Only supported one at the moment
+        python-version: ['3.10', '3.11'] # Only supported one at the moment
```

Comment:
nit: should this comment be dropped now (like above)?

```suggestion
        python-version: ['3.10', '3.11']
```

Reply:
Yes, I changed it locally but forgot to push.